workflow/tests/base_results/results_ashrae_140.csv

```diff
@@ -1,27 +1,27 @@
 Test Case,Annual Heating Load [MMBtu],Annual Cooling Load [MMBtu]
-L100AC.xml,51.41,N/A
-L110AC.xml,77.05,N/A
-L120AC.xml,39.84,N/A
-L130AC.xml,43.86,N/A
-L140AC.xml,48.05,N/A
-L150AC.xml,40.85,N/A
-L155AC.xml,44.14,N/A
-L160AC.xml,52.19,N/A
+L100AC.xml,62.0,N/A
+L110AC.xml,88.51,N/A
+L120AC.xml,50.2,N/A
+L130AC.xml,55.46,N/A
+L140AC.xml,61.35,N/A
+L150AC.xml,50.35,N/A
+L155AC.xml,54.51,N/A
+L160AC.xml,62.44,N/A
```

Comment:
This is concerning; I wouldn't expect any of the ASHRAE 140 test results to change. What caused them to change?

Reply:
I _think_ because these test files have zero occupants. Perhaps the year round vacancy period should only be applied when performing operational calculations.

Reply:
I tried running the 140 tests locally and got identical results between master and this branch. Where did these new numbers come from? I am confused.

Reply:
Maybe you ran the 140 tests _after_ I changed the vacancy period to only be applied when performing operational calculation.

Reply:
Ah that's it, thanks. I missed that change.